common/script_values/00_men_at_arms_values.txt

```diff
@@ -164,7 +170,7 @@ culture_ai_weight_skirmishers = {
 					has_innovation = innovation_snow_hunter
 					has_innovation = innovation_swarm
 					has_innovation = innovation_tidehunter
-					has_innovation = innovation_undead_ground
+					has_innovation = innovation_undead_tactics
```

Comment:
Old localization still there.
<details>
<summary>Click to expand</summary>

![image](https://user-images.githubusercontent.com/46384992/157839213-effd6340-becb-43a8-80b8-f7594c8fc814.png)

</details>